pkg/telemetryservice/sql/README.md

```diff
@@ -0,0 +1,22 @@
+## Shema for SQL
+create table before using telemetryService
+### MySQL
+```sql
+create table yourTableName (
+    ts timestamp,
+    rawData varchar(255)
+);
+```
+
+### SQLServer
+```sql
+create table yourTableName (
+    ts datetime2,
+    rawData varchar(255)
+)
+```
```

Comment:
This struct looks different from the design, which one is the latest? https://github.com/Edgenesis/shifu/pull/669/files

Reply:
The design should be the latest, sorry for ignoring that.